checker/src/org/checkerframework/checker/index/IndexUtils.java

```diff
@@ -0,0 +1,106 @@
+package org.checkerframework.checker.index;
+
+import com.sun.source.tree.Tree;
+import java.util.Collections;
+import java.util.List;
+import javax.lang.model.element.AnnotationMirror;
+import org.checkerframework.checker.index.minlen.MinLenAnnotatedTypeFactory;
+import org.checkerframework.checker.index.qual.MinLen;
+import org.checkerframework.common.value.ValueAnnotatedTypeFactory;
+import org.checkerframework.common.value.qual.IntVal;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+import org.checkerframework.javacutil.AnnotationUtils;
+
+/** A collection of utility functions used by several index checker subcheckers. */
+public class IndexUtils {
+
+    /**
+     * Used to get the list of array names that an annotation applies to. Can return null if the
```

Comment:
Change "used to get" to "gets".  That makes the writing more active -- and shorter.
The implementation doesn't seem to depend on array names.  Maybe you mean that if the argument is from the upperbound or samelen hierarchy, then the return value is a list of array names.